utils/utils.go

```diff
@@ -0,0 +1,9 @@
+package utils
```

Comment:
I think this data structure could be better located on a package called `model` instead of utils

Reply:
And the file this is in could be named `pokemon.go` that way we know that we're dealing with pokemon related models